app/models/configuration_script.rb

```diff
@@ -0,0 +1,3 @@
+class ConfigurationScript < ActiveRecord::Base
```

Comment:
@bdunne This is merged now, but I would still determine a better name for this class.  We are using it for Ansible JobTemplates but `ConfigurationScripts` makes me think of Playbooks.  

Maybe `ConfigurationJobTemplate`, but open to suggestions.
cc @syncrou